metabase/metabase
# ci(sdk): Fix can't build SDK from master

### Description

We unintentionally included tests files when building the SDK, so it failed the build

### How to verify

run `yarn build-embedding-sdk` and it shouldn't fail. You can try the same on `master` and it should fail.

This also rework https://github.com/metabase/metabase/pull/43242. By removing the tests files from TS, but correct the one file test that's accidentally imported.


### Demo
#### Before
![image](https://github.com/user-attachments/assets/9196ab14-9b8d-4e5e-9bbc-b13cb5626cbc)

#### After
![image](https://github.com/user-attachments/assets/6011b47d-ac2b-4a8a-a2a8-3db1f62ce75f)
